### ISSUE TITLE
Black linter "failed" though no issues detected 

### ISSUE THREAD

**Issue by rindPHI** [wearerequired/lint-action#513]
Hi there,

I'm using the black linter a GitHub action [in my project](https://github.com/rindPHI/isla) as follows:

```yml
name: Lint

on:
  # Trigger the workflow on push or pull request,
  # but only for the main branch
  push:
    branches:
      - main
  pull_request:
    branches:
      - main

jobs:
  run-linters:
    name: Run linters
    runs-on: ubuntu-latest

    steps:
      - name: Check out Git repository
        uses: actions/checkout@v2

      - name: Set up Python
        uses: actions/setup-python@v1
        with:
          python-version: "3.10"

      - name: Install Python dependencies
        run: pip install black

      - name: Run linters
        uses: wearerequired/lint-action@v2
        with:
          black: true
          black_dir: "src/"
          black_args: '-extend=exclude="(^/src/isla/bnf/* | ^/src/isla/isla_language/* | ^/src/isla/mexpr_lexer/* | ^/src/isla/mexpr_parser/*)"'
```

Everything seems to work, with one exception: Although the linter run reports "no issues," I get a "failed" feedback (red cross), see e.g. https://github.com/rindPHI/isla/runs/8190559827. I would rather not have a red cross at my project page next to its name though no issues were detected :) Can anybody help me with this?

Thanks in advance!

<img width="500" alt="No issues but red cross" src="https://user-images.githubusercontent.com/448160/188469660-30cdfcd0-2a7d-45eb-b622-bc388535160e.png">

**Comment by rindPHI** [comment]
Sorry, I found the issue: The parameter `-extend=exclude` was wrongly spelled, don't know how that happened. Sorry!
